### PR TITLE
Modified to use ghcr.io as a container registry

### LINKED ISSUE #106: Use ghcr.io for conatiner registry
## What

We now use quay.io for container registry, but it is boring to manage users and credentials.
Github provides a container registry service. It is free for public images and its access control is integrated with GitHub.

## How

- enable github container registry
- migrate all images from quay.io to ghcr.io
- rewrite image names on codes.

## Checklist

- [ ] Finish implementation of the issue
- [ ] Test all functions
- [ ] Have enough logs to trace activities
- [ ] Notify developers of necessary actions

### PR DESCRIPTION
**Pull request by bells17**
fix #106

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ spec:
 
 ## Container images
 
-Container images are available on [Quay.io](https://quay.io/repository/topolvm/pvc-autoresizer)
+Container images are available on [ghcr.io](https://github.com/topolvm/pvc-autoresizer/pkgs/container/pvc-autoresizer)  
+**Please note that [the image on quay.io](https://quay.io/repository/topolvm/pvc-autoresizer) is currently deprecated and will no longer be available in the future.**
```
Comment (toshipp) [inline README.md:115]:
```suggestion
**Please note that [the images on quay.io](https://quay.io/repository/topolvm/pvc-autoresizer) are currently deprecated and will no longer be available in the future.**
```

**Review by toshipp — CHANGES_REQUESTED** [review]

**Review by toshipp — DISMISSED** [review]
LGTM

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
           if [ $(echo $VERSION | grep "-") ]; then PRERELEASE=true; else PRERELEASE=false; fi
           echo "::set-output name=version::${VERSION}"
           echo "::set-output name=prerelease::${PRERELEASE}"
-      - run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USER }} --password-stdin quay.io
+      - run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ secrets.GHCR_USER }} --password-stdin
```
Comment (daichimukai) [inline .github/workflows/release.yaml:22]: @bells17 
What is `GHCR_USER`? One of GitHuB built-in secret (like `GITHUB_TOKEN`) or maintainers should set this secret?

Comment (bells17) [reply .github/workflows/release.yaml]: Sorry. I fixed it.
It seems that GITHUB_TOKEN is also set automatically when you allow deployment from Github Package to Github Actions.

- https://docs.github.com/en/actions/publishing-packages/publishing-docker-images
- https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility
- https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

**Review by toshipp — APPROVED** [review]

**Review by daichimukai — APPROVED** [review]
LGTM

**Pull request merged**